test/utils/check_rpm_install.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
```suggestion
#!/usr/bin/env python3
```
All my comments may fail if we need to support python2

Reply:
I had to do the opposite when updating `systemctl.py`, it seems the image doesn't provide a `python3` executable, only a versionless `python`